changelogs/unreleased/6013-davinci26-small.md

```diff
@@ -0,0 +1,4 @@
+## Support for Global Circuit Breaker Policy
+
+The way [circuit-breaker-annotations](https://projectcontour.io/docs/1.27/config/annotations/) work currently is that when not present they are being defaulted to Envoy defaults. The Envoy defaults can be quite low for larger clusters with more traffic so if
+a user accidentally deletes them or unset them this cause an issue. With this change we are providing contour administrators the ability to provide global defaults that are good. In that case even if the user forgets to set them or deletes them they can have the safety net of good defaults.
```

Comment:
Let's mention that these are set in the config file via `cluster.circuit-breakers`, and add a link to https://projectcontour.io/docs/1.28/configuration/#circuit-breakers release, or via ContourConfiguration CRD in `spec.envoy.cluster.circuitBreakers` with a link to https://projectcontour.io/docs/1.28/config/api/#projectcontour.io/v1alpha1.GlobalCircuitBreakerDefaults

It's fine that the links are broken right now, they'll resolve when the release is cut.

Reply:
fixed in 4151035aa07d44ff5b162b52dd73935da4f7cda2